### DISCUSSION TITLE
Is it possible to use time/date data?

### DISCUSSION THREAD

**Discussion by ofenloch** [alandefreitas/matplotplusplus#383] [Q&A] [answered]
Hi, everybody!
I'm trying to figure out how to plot a time series.
In Gnuplot I'd do something like this

     set timefmt "%Y-%m-%d"
     set format x "%Y-%m-%d"
     set xtics 60*60*24*30*12*10
     plot DATA_FILE using 1:3 with lines linetype 2 linewidth 1

and get something like this

![image](https://github.com/alandefreitas/matplotplusplus/assets/57812959/7d5c3d20-7af6-441b-9a93-d9eca2845701)

Can I do this with Matplot++, too?

Regards

Oliver

**Comment by alandefreitas** [comment, marked as answer]
I believe there's no option for time series in the library.
